README.md

```diff
@@ -34,22 +34,23 @@ This is how you can use colin afterwards:
 
 ```
 $ colin -h
-Usage: colin [OPTIONS] TARGET
+Usage: colin [OPTIONS] COMMAND [ARGS]...
+
+  COLIN -- Container Linter
 
 Options:
-  -c, --config [redhat|fedora]  Select a predefined configuration.
-  --debug                       Enable debugging mode (debugging logs, full tracebacks).
-  -f, --config-file FILENAME    Path to a file to use for validation (by
-                                default they are placed in /usr/share/colin).
-  --json FILENAME               File to save the output as json to.
```

Comment:
Why did you remove --json option?

Reply:
The `--json` is only valid for subcommands. (`check`/`list-checks`)